sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppDestination.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.sample.compose.ui
+
+import androidx.navigation.*
+
+const val KEY_ID = "id"
+
+sealed class AppDestination(val route: String = "") {
+
+    open val arguments: List<NamedNavArgument> = emptyList()
+
+    open var destination: String = route
+
+    //====================================================//
+
+    object Up : AppDestination()
+
+    object Home : AppDestination("home")
+
+    /**
+     * We can define route as "coin/details" without "coinId" parameter because we're passing it as argument already.
+     * So either passing "coinId" via arguments or passing it via route.
+     *
+     * We keep passing "coinId" in both route and arguments for this destination to give example of navigation concept
+     * about how to build a destination with parameters.
+     */
```

Comment:
Please update the description a bit  -->  just replace  some words of `"coin/details"` and `"coinId"`

Reply:
@Wadeewee I think we can remove this comment now 👍

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/367/commits/292e3157b647217bffd88bcba8b6874063dc981e